src/views/product-root-docs-path-landing/types.ts

```diff
@@ -0,0 +1,28 @@
+import { MDXRemoteSerializeResult } from 'next-mdx-remote'
+import { ProductData } from 'types/products'
+
+interface PageContent {
+  pageSubtitle: string
+  // TODO create a block type
```

Comment:
💭 There's some typing and validation work in the `product-landing` view, if we want to go the Joi + types route ☺️

Reply:
I have high hopes that we can create a single, robust system for filling in content on all landing pages 😍 (definitely future ha!)